nrf52810-hal/memory.x

```diff
@@ -0,0 +1,23 @@
+/* Linker script for the nRF52 - WITHOUT SOFT DEVICE */
```

Comment:
Does the 810 come in different packages with different RAM/Flash amounts? Or is there only one configuration?

Reply:
There is only one configuration